pulp_rpm/tests/functional/api/test_pulpimport.py

```diff
@@ -190,6 +175,32 @@ def _perform_import(self, importer, chunked=False):
 
         return task_group
 
+
+class PulpImportTestCase(PulpImportTestBase):
+    """
+    Basic tests for PulpImporter and PulpImport.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables."""
+        cls.cfg = config.get_config()
+        cls.client = api.Client(cls.cfg, api.json_handler)
+        cls.core_client = CoreApiClient(configuration=cls.cfg.get_bindings_config())
+        cls.rpm_client = gen_rpm_client()
+
+        cls.repo_api = RepositoriesRpmApi(cls.rpm_client)
+        cls.remote_api = RemotesRpmApi(cls.rpm_client)
+        cls.exporter_api = ExportersPulpApi(cls.core_client)
+        cls.exports_api = ExportersCoreExportsApi(cls.core_client)
+        cls.importer_api = ImportersPulpApi(cls.core_client)
+        cls.imports_api = ImportersCoreImportsApi(cls.core_client)
+
+        (cls.import_repos, cls.export_repos, cls.remotes) = cls._setup_repositories()
```

Comment:
@daviddavis, just curious why to use parenthesis here? Is it some kind of convention or just your preference?

Reply:
I didn't write this line. It comes from [here](https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/tests/functional/api/test_pulpimport.py#L114). I don't have a preference either way though.